File: python/problem1131.py
inter = 0
gremio = 0
empate = 0
jogos = 0
while True:
    jogos += 1
    x, y = map(int, input().split())
    if x > y:
        inter += 1
    elif x < y:
        gremio += 1
    else:
        empate += 1
    print('Novo grenal (1-sim 2-nao)')
    choice = int(input())
    if choice != 1:
        break

print('%d grenais' % jogos)
print(f'Inter:{inter}')
print(f'Gremio:{gremio}')
print(f'Empates:{empate}')
if inter > gremio:
    print('Inter venceu mais')
elif gremio > inter:
    print('Gremio venceu mais')
else:
    print('Não houve vencedor')

File: java/Problem1131.java
import java.util.Scanner;

public class Problem1131 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int inter = 0;
        int gremio = 0;
        int empates = 0;
        int grenais = 0;
        int opt = 1;
        while (opt == 1) {
            grenais++;
            int gInter = sc.nextInt();
            int gGremio = sc.nextInt();
            if (gGremio > gInter) {
                gremio++;
            }
            if (gGremio < gInter) {
                inter++;
            }
            if (gGremio == gInter) {
                empates++;
            }

            System.out.println("Novo grenal (1-sim 2-nao)");
            opt = sc.nextInt();
        }

        System.out.printf("%d grenais\n", grenais);
        System.out.printf("Inter:%d\n", inter);
        System.out.printf("Gremio:%d\n", gremio);
        System.out.printf("Empates:%d\n", empates);

        if (gremio > inter) {
            System.out.println("Gremio venceu mais");
        } 
        if (gremio < inter) {
            System.out.println("Inter venceu mais");
        }
        if (gremio == inter) {
            System.out.println("Não houve vencedor");
        }

        sc.close();
    }
}